mutation_partition.hh

```diff
@@ -903,6 +903,16 @@ class rows_entry final : public evictable {
     intrusive_b::member_hook _link;
     clustering_key _key;
     deletable_row _row;
+
+    // Given p is the preceding rows_entry&,
+    // this tombstone applies to the range (p.position(), position()] if continuous()
+    // and to [position(), position()] if !continuous().
+    // So the tombstone applies only to the continuous interval, to the left.
+    // On top of that, _row.deleted_at() may still apply new information.
+    // So it's not deoverlapped with the row tombstone.
```

Comment:
Is "not deoverlapped" true even if the row tombstone is older than the range tombstone?

Or is it true only in some circumstances?

Reply:
It's not guaranteed that it's deoverlapped even if the row tombstone is older. 

In some cases they will be, we compact rows covered by incoming range tombstones in apply(), but it's not guaranteed in general, e.g. if older row tombstone arrives after the range tombstone.